components/discover/defiTable.tsx

```diff
@@ -249,10 +295,10 @@ export const columns: ColumnDef<TableInfo>[] = [
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const DataTable: FunctionComponent<DataTableProps> = ({ data, loading }) => {
   const [sorting, setSorting] = useState<SortingState>([
-    {
-      id: "apr",
-      desc: true,
-    },
+    // {
```

Comment:
Aaah my bad!! Sorry, I didn't know the APR field was selected by default, that's why the hover effect was already present.
So you can bring this default behaviour back, now I understand why it was highlighted by default